storefront/app/view_models/workarea/storefront/product_view_model.rb

```diff
@@ -18,12 +17,12 @@ def breadcrumbs
           if options[:via].present?
             Navigation::Breadcrumbs.from_global_id(
               options[:via],
-              last: model.name
+              last: model
```

Comment:
So why do this unnecessary change? Better to reduce churn for upgrades if this doesn't affect anything.

Reply:
this isn't unnecessary, it's what fixes the schema

Reply:
again, this isn't unnecessary. i wrote in an above conversation:

> that would kinda defeat the purpose of all this, because a string isn't `.navigable?`, so we still need to pass the full product model into the breadcrumbs as `:last` in order to generate the right URL.

basically, i need `:last` to be a model in order for the Schema.org helper methods to render the right URL for the last Taxon.